src/components/RadioButton/RadioButton.spec.tsx

```diff
@@ -24,16 +24,40 @@ describe('RadioButton', () => {
 
   it('should show checked state', async () => {
     const driver = createDriver(<RadioButton checked {...defProps} />);
-
     expect(await driver.isChecked()).toBeTruthy();
   });
 
   it('should show disabled state', async () => {
     const driver = createDriver(<RadioButton disabled {...defProps} />);
-
     expect(await driver.isDisabled()).toBeTruthy();
   });
 
+  it('should have focus state', async () => {
+    const driver = createDriver(<RadioButton {...defProps} />);
+    await driver.clickInput();
+    expect(await driver.isFocused()).toBeTruthy();
+  });
+
+  it('should have focus ring with withFocusRing prop', async () => {
+    const driver = createDriver(<RadioButton withFocusRing {...defProps} />);
+    await driver.clickInput();
+    expect(await driver.isContainsFocusRing()).toBeTruthy();
+  });
+
+  it('should not have focus ring without prop', async () => {
+    const driver = createDriver(<RadioButton {...defProps} />);
+    await driver.clickInput();
+    expect(await driver.isContainsFocusRing()).toBeFalsy();
+  });
+
+  it('should not have focus ring after blur', async () => {
+    const driver = createDriver(<RadioButton withFocusRing {...defProps} />);
+    await driver.clickInput();
+    expect(await driver.isContainsFocusRing()).toBeTruthy();
+    await driver.blurInput();
+    expect(await driver.isContainsFocusRing()).toBeFalsy();
+  });
```

Comment:
I would move these tests to an `e2e` file.
Instead of checking that the focus ring exists, we should just take a screenshot.

you can use `eyes.it` for this purpose.
You can see an example in [AddItem.e2e.ts](https://github.com/wix/wix-ui-tpa/blob/master/src/components/AddItem/AddItem.e2e.ts)
For the RadioButton, you'll need to use the data-hook from [here](https://github.com/wix/wix-ui-tpa/blob/36f8a066d5a54a7d4a0a34018bbf8d81ec6efb07/src/components/RadioButton/docs/index.story.tsx#L47)

Reply:
you can read more about eyes.it [here](https://github.com/wix/eyes.it)